### ISSUE TITLE
(Beta) Match Error - Cyborg 009 vs. Devilman

### ISSUE THREAD

**Issue by sven-7** [ZeroQI/Hama.bundle#139]
I've Plex dance'd this show a few times, but it still shows up as Cyborg 009 (2001) instead of Cyborg 009 vs. Devilman.  If I force re-match in Plex, it stays the same. Not sure what is happening here.

[com.plexapp.agents.hama.zip](https://github.com/ZeroQI/Hama.bundle/files/1261421/com.plexapp.agents.hama.zip)

**Comment by ZeroQI** [comment]
side note, corrected: 2017-08-29 15:00:13,450 (2158) :  ERROR (logkit:22) - DefaultPrefs.json, Value 'Simkl' missing, update+save so it list all Pres in DefaultPrefs.json

AniDB titles in Plex Media Server\Plug-in Support\Data\com.plexapp.agents.hama\DataItems\AniDB\anime-titles.xml
```
	<anime aid="11169">
		<title type="official" xml:lang="ja">サイボーグ009VSデビルマン</title>
		<title type="syn" xml:lang="ja">デビルマン (2015)</title>
		<title type="main" xml:lang="x-jat">Cyborg 009 vs. Devilman</title>
		<title xml:lang="x-jat" type="syn">Devilman (2015)</title>
		<title xml:lang="zh-Hant" type="syn">人造人009VS惡魔人</title>
	</anime>
```

AniDB.GetMetadata() - 'title': Cyborg 009 vs. Devilman, 'title_sort': ('Cyborg 009 vs. Devilman', 'Cyborg 009 vs. Devilman'), original_title: Cyborg 009 vs. Devilman

During the field per field summary at the end of hama log, it seems Title meta source order is 'TheTVDB' only and the tvdb title is in the xml: [TheTVDB xml for 72745](http://thetvdb.plexapp.com/api/A27AD9BE0DA63333/series/72745/all/en.xml
)
`<SeriesName>Cyborg 009 (2001)</SeriesName>`

2017-08-29 15:00:14,002 (d24) :  INFO (logkit:16) - [1] title                          Type: str                   Source: TheTVDB      Value: 'Cyborg 009 (2001)'

The title is shared by both episodes and series. Will separate that shortly
title is taken on tvdb only and that's the tvdb title...
ou may want "AniDB, TheTVDB" for titles

**Comment by sven-7** [comment]
Ah. So TVDB treats 'Cyborg 009 vs. Devilman' as a special attached to 'Cyborg 009 (2001)', while AniDB has it listed differently. I suppose I can just manually rename the title.

**Comment by ZeroQI** [comment]
anidb id assigned to serie but it maps to this tvdb serie since it's a special and has that name...

**Comment by sven-7** [comment]
Yep, no big deal. I hoped adding a anidb.id might force it to use anidb titles, but no big deal. I tried to get TVDB to separate it (it really doesn't belong there), but because it's only three episodes they don't consider it enough to be a separate series.

**Comment by sven-7** [comment]
Closed #139

**Comment by ZeroQI** [comment]
Anidb.id would force the id
The title come from the metadata source in the agent settings regardless of the id. 
I would use AniDB for the title in the order 'main, en, jap'... if you put in title filed priority a '|' you can specify the priority for series title then after the pipe char the ep metadata sources priority
